src/views/TradingCompetition/components/CompetitionHeadingText.tsx

```diff
@@ -0,0 +1,25 @@
+import { Heading } from '@pancakeswap-libs/uikit'
+import styled from 'styled-components'
+
+const sharedStyles = `
+font-size: 32px;
```

Comment:
There are a few things about the heading text on the Competition page that differ from our uikit

1. The H1s are 64px on Desktop, 44px on Mobile
2. The H2s are 40px on Desktop, 32px on Mobile
3. They all make use of some gradient stroke stiles. 

I'll manage all of this in the FE for now and then we can see about how to translate them over to the uikit & our existing Headings

<img width="797" alt="Screenshot 2021-03-19 at 08 09 54" src="https://user-images.githubusercontent.com/79279477/111812832-62608580-88d0-11eb-9142-5f4f499a7938.png">

Reply:
See with Bugs if it's really needed. It's better to stick to the uikit rules as much as we can